changelog/add-overview-active-disputes-notice

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Improve the wording and style of the "Active Disputes" task list item on the Payments → Overview screen to better communicate the urgency of resolving these disputes.
```

Comment:
Since we're now only showing disputes due within 7d, should we hint that in the changelog?

It's too crowded to add it in this changelog entry, so I'm suggesting to add a new changelog entry.

Feel free to come up with better wording but I'm thinking:
```
Show only disputes waiting for response that are due within 7 days on the Payments → Overview screen.
```

Reply:
Thanks! Good idea, I've added another changelog entry in 3209294:

Display the "Active Disputes" task list item on the Payments → Overview screen only if there are disputes due within seven days.